tests/test_trr.py

```diff
@@ -219,6 +219,18 @@ def test_ragged_2():
             f.write(xyz)
 
 
+def test_malformed_trr():
+    with open(temp, 'w') as tmpf:
+        tmpf.write("foo")  # verty badly malformed TRR
+
+    with pytest.raises(OSError):
+        TRRTrajectoryFile(temp)
+
+    psutil = pytest.importorskip("psutil")
+    open_files = psutil.Process().open_files()
+    assert open_files == []
```

Comment:
What if pytest is writing to a XML results file while running this assertion? I think this has to be more specific.